ush/generate_FV3LAM_wflow.py

```diff
@@ -397,10 +394,9 @@ def generate_FV3LAM_wflow():
             #
             # Parameters that determine the set of cycles to run.
             #
-            "date_first_cycl": date_to_str(DATE_FIRST_CYCL, format="%Y%m%d"),
-            "date_last_cycl": date_to_str(DATE_LAST_CYCL, format="%Y%m%d"),
-            "cdate_first_cycl": cdate_first_cycl,
-            "cycl_hrs": cycl_hrs_str,
+            "date_first_cycl": date_to_str(DATE_FIRST_CYCL, format="%Y%m%d%H00"),
+            "date_last_cycl": date_to_str(DATE_LAST_CYCL, format="%Y%m%d%H00"),
```

Comment:
This is where the trailing zeros are added (mentioned above).

Reply:
One thing I like about this new approach is that I have had difficulty maintaining leading 0s in `CYCLE_HRS`, as you can see in the yaml config files, and has caused me subtle bugs. Now that it is part of the date, there is no doubt it will have leading zeros for single digit cycles.